dbt/adapters/sqlserver/connections.py

```diff
@@ -159,55 +236,53 @@ def open(cls, connection):
                 elif type_auth == "ActiveDirectoryMsi":
                     raise ValueError("ActiveDirectoryMsi is not supported yet")
 
-            elif type_auth == "ServicePrincipal":
-                app_id = getattr(credentials, "AppId", None)
-                app_secret = getattr(credentials, "AppSecret", None)
-
             elif getattr(credentials, "windows_login", False):
-                con_str.append(f"trusted_connection=yes")
+                con_str.append("trusted_connection=yes")
             elif type_auth == "sql":
-                #con_str.append("Authentication=SqlPassword")
+                con_str.append("Authentication=SqlPassword")
```

Comment:
Ditto here about the difference with `dbt-synapse`.

Reply:
again another thing we should address with #65. Also check out https://github.com/dbt-msft/dbt-sqlserver/pull/62#issuecomment-736183924 for the long history of the `SQLPassword`. what a headache.